Reporting causes NewPipe to crash

<!--
Oh no, a bug! It happens. Thanks for reporting an issue with NewPipe. To make it easier for us to help you please enter detailed information in the template we have provided below. If a section isn't relevant, just delete it, though it would be helpful to still provide as much detail as possible.
-->

<!-- IF YOU DON'T FILL IN THE TEMPLATE PROPERLY, YOUR ISSUE IS LIABLE TO BE CLOSED. If you feel tired/lazy right now, open your issue some other time. We'll wait. -->

<!-- The comments between these brackets won't show up in the submitted issue (as you can see in the Preview). -->

### Checklist
<!-- The first box has been checked for you to show you how it is done. -->

- [x] I am using the latest version - 0.20.4 <!-- Check https://github.com/TeamNewPipe/NewPipe/releases -->
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. <!-- Seriously, check. O_O -->
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug
<!--
1. Go to '...'
2. Press on '....'
3. Swipe down to '....'
-->

1. Go to any video where the comments fail to load.
(Or repeatedly switch videos quickly, you'll certainly get a video where comments don't load)
2. A little pop-up says "Sorry, something went wrong. <b>Report</b>."
3. Clicking Report causes the app to crash.

Could reproduce almost 70% of the time.

<!-- If you can't cause the bug to show up again reliably (and hence don't have a proper set of steps to give us), please still try to give as many details as possible on how you think you encountered the bug. -->



### Actual behaviour
<!-- Tell us what happens with the steps given above. -->

NewPipe crashes without any crash report and goes into a loop of showing the splash screen and crashing.

### Expected behavior
<!-- Tell us what you expect to happen. -->

I haven't actually used that report button before encountering this issue.
Expected behaviour: I guess app shouldn't crash

### Screenshots/Screen recordings
<!-- If applicable, add screenshots or a screen recording to help explain your problem. GitHub supports uploading them directly in the issue text box. If your file is too big for Github to accept, feel free to paste a link from an image/video hoster here instead. -->

<!-- DON'T POST SCREENSHOTS OF THE ERROR PAGE. Use the buttons given on the error page to paste the error as text in the Logs section below. -->

![Screenshot_20201123-190704](https://user-images.githubusercontent.com/65857597/99980103-dd57da80-2dcd-11eb-934e-f567fb9518fb.png)
This report button at bottom right.

Video in the screenshot: 
https://www.youtube.com/watch?v=5GSeWdjyr1c
### Logs
<!-- If your bug includes a crash (where you're shown the Error Report page with a bunch of info), tap on "Copy formatted report" at the bottom and paste it here: -->

<!-- That's right, here! -->

<details>
<summary>
<b>Logcat</b>
</summary>
<p>

```
11-23 19:03:42.791 I/Timeline(32688): Timeline: Activity_launch_request id:org.schabi.newpipe time:325274613
11-23 19:03:42.801 E/Parcel  (2906): Class not found when unmarshalling: org.schabi.newpipe.report.ErrorInfo
11-23 19:03:42.801 E/Parcel  (2906): java.lang.ClassNotFoundException: org.schabi.newpipe.report.ErrorInfo
11-23 19:03:42.801 E/Parcel  (2906): Caused by: java.lang.ClassNotFoundException: org.schabi.newpipe.report.ErrorInfo
11-23 19:03:42.811 V/WindowManager(2906): addAppToken: AppWindowToken{24ddbd7 token=Token{e5cbf56 ActivityRecord{3d91571 u0 org.schabi.newpipe/.report.ErrorActivity t1710}}} to stack=1 task=1710 at 1
11-23 19:03:42.841 D/GameManagerService(2906): NotifyRunnable. pkg: org.schabi.newpipe, type: 4, isMinimized: false, isTunableApp: false
11-23 19:03:42.846 V/MARsPolicyManager(2906): updatePackagesScore PackageInfo name -- org.schabi.newpipe
11-23 19:03:42.966 E/ACRA    (32688): ACRA caught a IllegalAccessError for org.schabi.newpipe
11-23 19:03:42.966 E/ACRA    (32688): java.lang.IllegalAccessError: Method 'void org.schabi.newpipe.report.ErrorInfo.<init>(org.schabi.newpipe.report.UserAction, java.lang.String, java.lang.String, int)' is inaccessible to class 'org.schabi.newpipe.report.ErrorInfo$Creator' (declaration of 'org.schabi.newpipe.report.ErrorInfo$Creator' appears in /data/app/org.schabi.newpipe-1/base.apk)
11-23 19:03:42.966 E/ACRA    (32688): 	at org.schabi.newpipe.report.ErrorInfo$Creator.createFromParcel(Unknown Source)
11-23 19:03:42.966 E/ACRA    (32688): 	at org.schabi.newpipe.report.ErrorActivity.onCreate(ErrorActivity.java:209)
11-23 19:03:43.401 I/WindowManager(2906): Screenshot max retries 4 of Token{e5cbf56 ActivityRecord{3d91571 u0 org.schabi.newpipe/.report.ErrorActivity t1710 f}} appWin=Window{ef41a93 u0 d0 p32688 org.schabi.newpipe/org.schabi.newpipe.MainActivity} drawState=4
11-23 19:03:43.586 I/WindowState(2906): WIN DEATH: Window{ef41a93 u0 d0 p32688 org.schabi.newpipe/org.schabi.newpipe.MainActivity}
11-23 19:03:43.586 I/ActivityManager(2906): Process org.schabi.newpipe (pid 32688)(adj 0) has died(109,242)
11-23 19:03:43.591 D/ActivityManager(2906): removeProcessNameLocked mProcessNames.remove pid=32688 ,hash=23487259 ,name=org.schabi.newpipe
11-23 19:03:43.861 D/PackageManager(2906): getComponentMetadataForIconTray : org.schabi.newpipe.MainActivity does not exist in mServices
11-23 19:03:43.861 D/PackageManager(2906): getComponentMetadataForIconTray : org.schabi.newpipe.MainActivity does not exist in mProviders
11-23 19:03:43.861 D/PackageManager(2906): getComponentMetadataForIconTray : org.schabi.newpipe.MainActivity does not exist in mReceivers
11-23 19:03:43.866 I/ApplicationPackageManager(3274): load=org.schabi.newpipe, bg=96-96, dr=96-96
11-23 19:03:44.476 D/MountService(2906): getExternalStorageMountMode : final mountMode=3, uid : 10753, packageName : org.schabi.newpipe
11-23 19:03:44.491 I/ActivityManager(2906): Start proc 333:org.schabi.newpipe:acra/u0a753 for service org.schabi.newpipe/org.acra.sender.JobSenderService
11-23 19:03:44.491 W/SELinux (333): SELinux: seapp_context_lookup: seinfo=default, level=s0:c512,c768, pkgname=org.schabi.newpipe:acra 
11-23 19:03:44.551 W/System  (333): ClassLoader referenced unknown path: /data/app/org.schabi.newpipe-1/lib/arm
11-23 19:03:45.006 I/ACRA    (333): Sending report /data/user/0/org.schabi.newpipe/app_ACRA-approved/2020-11-23T19:03:43.036+05:30.stacktrace
11-23 19:03:45.016 I/Timeline(333): Timeline: Activity_launch_request id:org.schabi.newpipe time:325276839
11-23 19:03:45.021 E/Parcel  (2906): Class not found when unmarshalling: org.schabi.newpipe.report.ErrorInfo
11-23 19:03:45.021 E/Parcel  (2906): java.lang.ClassNotFoundException: org.schabi.newpipe.report.ErrorInfo
11-23 19:03:45.021 E/Parcel  (2906): Caused by: java.lang.ClassNotFoundException: org.schabi.newpipe.report.ErrorInfo
11-23 19:03:45.026 I/ActivityManager(2906): isWidgetUsingPkg : org.schabi.newpipe no widget is using.
11-23 19:03:45.031 V/WindowManager(2906): addAppToken: AppWindowToken{ae768c7 token=Token{c1aa06 ActivityRecord{2493ce1 u0 org.schabi.newpipe/.report.ErrorActivity t1711}}} to stack=1 task=1711 at 0
11-23 19:03:45.041 D/MountService(2906): getExternalStorageMountMode : final mountMode=3, uid : 10753, packageName : org.schabi.newpipe
11-23 19:03:45.056 W/SELinux (355): SELinux: seapp_context_lookup: seinfo=default, level=s0:c512,c768, pkgname=org.schabi.newpipe 
11-23 19:03:45.061 I/ActivityManager(2906): Start proc 355:org.schabi.newpipe/u0a753 for activity org.schabi.newpipe/.report.ErrorActivity
11-23 19:03:45.076 V/WindowStateAnimator(2906): Finishing drawing window Window{dee9bd5 u0 d0 p2906 Starting org.schabi.newpipe}: mDrawState=DRAW_PENDING
11-23 19:03:45.101 D/ActivityManager(2906):  Launching org.schabi.newpipe, updated priority
11-23 19:03:45.111 D/GameManagerService(2906): NotifyRunnable. pkg: org.schabi.newpipe, type: 4, isMinimized: false, isTunableApp: false
11-23 19:03:45.126 V/MARsPolicyManager(2906): updatePackagesScore PackageInfo name -- org.schabi.newpipe
11-23 19:03:45.131 D/GameManagerService(2906): identifyGamePackage. org.schabi.newpipe
11-23 19:03:45.131 D/StatusBarManagerService(2906): manageDisableList userId=0 what=0x0 pkg=Window{dee9bd5 u0 d0 p2906 Starting org.schabi.newpipe}
11-23 19:03:45.131 D/GameManagerService(2906): identifyGamePackage. org.schabi.newpipe
11-23 19:03:45.131 D/GameManagerService(2906): identifyGamePackage. org.schabi.newpipe
11-23 19:03:45.146 W/System  (355): ClassLoader referenced unknown path: /data/app/org.schabi.newpipe-1/lib/arm
11-23 19:03:45.226 I/ACRA    (355): ACRA is enabled for org.schabi.newpipe, initializing...
11-23 19:03:45.896 E/ACRA    (355): ACRA caught a IllegalAccessError for org.schabi.newpipe
11-23 19:03:45.896 E/ACRA    (355): java.lang.IllegalAccessError: Method 'void org.schabi.newpipe.report.ErrorInfo.<init>(org.schabi.newpipe.report.UserAction, java.lang.String, java.lang.String, int)' is inaccessible to class 'org.schabi.newpipe.report.ErrorInfo$Creator' (declaration of 'org.schabi.newpipe.report.ErrorInfo$Creator' appears in /data/app/org.schabi.newpipe-1/base.apk)
11-23 19:03:45.896 E/ACRA    (355): 	at org.schabi.newpipe.report.ErrorInfo$Creator.createFromParcel(Unknown Source)
11-23 19:03:45.896 E/ACRA    (355): 	at org.schabi.newpipe.report.ErrorActivity.onCreate(ErrorActivity.java:209)
11-23 19:03:46.306 I/WindowManager(2906): Screenshot max retries 4 of Token{c1aa06 ActivityRecord{2493ce1 u0 org.schabi.newpipe/.report.ErrorActivity t1711 f}} appWin=Window{dee9bd5 u0 d0 p2906 Starting org.schabi.newpipe} drawState=4
11-23 19:03:46.321 D/PackageManager(2906): getComponentMetadataForIconTray : org.schabi.newpipe.report.ErrorActivity does not exist in mServices
11-23 19:03:46.321 D/PackageManager(2906): getComponentMetadataForIconTray : org.schabi.newpipe.report.ErrorActivity does not exist in mProviders
11-23 19:03:46.321 D/PackageManager(2906): getComponentMetadataForIconTray : org.schabi.newpipe.report.ErrorActivity does not exist in mReceivers
11-23 19:03:46.321 I/ApplicationPackageManager(3274): load=org.schabi.newpipe, bg=96-96, dr=96-96
11-23 19:03:46.446 I/ActivityManager(2906): Process org.schabi.newpipe (pid 355)(adj 0) has died(96,242)
11-23 19:03:46.446 D/ActivityManager(2906): removeProcessNameLocked mProcessNames.remove pid=355 ,hash=231829482 ,name=org.schabi.newpipe
11-23 19:03:47.421 I/ACRA    (333): Sending report /data/user/0/org.schabi.newpipe/app_ACRA-approved/2020-11-23T19:03:45.939+05:30.stacktrace
11-23 19:03:47.431 I/Timeline(333): Timeline: Activity_launch_request id:org.schabi.newpipe time:325279252
11-23 19:03:47.436 E/Parcel  (2906): Class not found when unmarshalling: org.schabi.newpipe.report.ErrorInfo
11-23 19:03:47.436 E/Parcel  (2906): java.lang.ClassNotFoundException: org.schabi.newpipe.report.ErrorInfo
11-23 19:03:47.436 E/Parcel  (2906): Caused by: java.lang.ClassNotFoundException: org.schabi.newpipe.report.ErrorInfo
11-23 19:03:47.436 I/ActivityManager(2906): isWidgetUsingPkg : org.schabi.newpipe no widget is using.
11-23 19:03:47.446 V/WindowManager(2906): addAppToken: AppWindowToken{14f2445 token=Token{2d94cbc ActivityRecord{17f80af u0 org.schabi.newpipe/.report.ErrorActivity t1712}}} to stack=1 task=1712 at 0
11-23 19:03:47.456 D/MountService(2906): getExternalStorageMountMode : final mountMode=3, uid : 10753, packageName : org.schabi.newpipe
11-23 19:03:47.471 I/ActivityManager(2906): Start proc 404:org.schabi.newpipe/u0a753 for activity org.schabi.newpipe/.report.ErrorActivity
11-23 19:03:47.476 W/SELinux (404): SELinux: seapp_context_lookup: seinfo=default, level=s0:c512,c768, pkgname=org.schabi.newpipe 
11-23 19:03:47.491 V/WindowStateAnimator(2906): Finishing drawing window Window{37a96fd u0 d0 p2906 Starting org.schabi.newpipe}: mDrawState=DRAW_PENDING
11-23 19:03:47.511 D/ActivityManager(2906):  Launching org.schabi.newpipe, updated priority
11-23 19:03:47.521 D/GameManagerService(2906): NotifyRunnable. pkg: org.schabi.newpipe, type: 4, isMinimized: false, isTunableApp: false
11-23 19:03:47.531 V/MARsPolicyManager(2906): updatePackagesScore PackageInfo name -- org.schabi.newpipe
11-23 19:03:47.531 D/StatusBarManagerService(2906): manageDisableList userId=0 what=0x0 pkg=Window{37a96fd u0 d0 p2906 Starting org.schabi.newpipe}
11-23 19:03:47.551 W/System  (404): ClassLoader referenced unknown path: /data/app/org.schabi.newpipe-1/lib/arm
11-23 19:03:47.621 I/ACRA    (404): ACRA is enabled for org.schabi.newpipe, initializing...
11-23 19:03:48.306 E/ACRA    (404): ACRA caught a IllegalAccessError for org.schabi.newpipe
11-23 19:03:48.306 E/ACRA    (404): java.lang.IllegalAccessError: Method 'void org.schabi.newpipe.report.ErrorInfo.<init>(org.schabi.newpipe.report.UserAction, java.lang.String, java.lang.String, int)' is inaccessible to class 'org.schabi.newpipe.report.ErrorInfo$Creator' (declaration of 'org.schabi.newpipe.report.ErrorInfo$Creator' appears in /data/app/org.schabi.newpipe-1/base.apk)
11-23 19:03:48.306 E/ACRA    (404): 	at org.schabi.newpipe.report.ErrorInfo$Creator.createFromParcel(Unknown Source)
11-23 19:03:48.306 E/ACRA    (404): 	at org.schabi.newpipe.report.ErrorActivity.onCreate(ErrorActivity.java:209)
11-23 19:03:48.691 I/WindowManager(2906): Screenshot max retries 4 of Token{2d94cbc ActivityRecord{17f80af u0 org.schabi.newpipe/.report.ErrorActivity t1712 f}} appWin=Window{37a96fd u0 d0 p2906 Starting org.schabi.newpipe} drawState=4
11-23 19:03:48.711 D/PackageManager(2906): getComponentMetadataForIconTray : org.schabi.newpipe.report.ErrorActivity does not exist in mServices
11-23 19:03:48.711 D/PackageManager(2906): getComponentMetadataForIconTray : org.schabi.newpipe.report.ErrorActivity does not exist in mProviders
11-23 19:03:48.711 D/PackageManager(2906): getComponentMetadataForIconTray : org.schabi.newpipe.report.ErrorActivity does not exist in mReceivers
11-23 19:03:48.716 I/ApplicationPackageManager(3274): load=org.schabi.newpipe, bg=96-96, dr=96-96
11-23 19:03:48.821 I/ActivityManager(2906): Process org.schabi.newpipe (pid 404)(adj 1) has died(94,242)
11-23 19:03:48.821 D/ActivityManager(2906): removeProcessNameLocked mProcessNames.remove pid=404 ,hash=165777394 ,name=org.schabi.newpipe
11-23 19:03:49.816 I/ACRA    (333): Sending report /data/user/0/org.schabi.newpipe/app_ACRA-approved/2020-11-23T19:03:48.338+05:30.stacktrace
11-23 19:03:49.826 I/Timeline(333): Timeline: Activity_launch_request id:org.schabi.newpipe time:325281646
11-23 19:03:49.831 E/Parcel  (2906): Class not found when unmarshalling: org.schabi.newpipe.report.ErrorInfo
11-23 19:03:49.831 E/Parcel  (2906): java.lang.ClassNotFoundException: org.schabi.newpipe.report.ErrorInfo
11-23 19:03:49.831 E/Parcel  (2906): Caused by: java.lang.ClassNotFoundException: org.schabi.newpipe.report.ErrorInfo
11-23 19:03:49.831 I/ActivityManager(2906): isWidgetUsingPkg : org.schabi.newpipe no widget is using.
```
</p>
</details>

<!-- Please fill this out when you do not provide a log generate by NewPipe -->

### Device info

 - Android version/Custom ROM version: Android 6.0.1
 - Device model: 
Samsung SM-G550FY

--- a/app/src/main/java/org/schabi/newpipe/report/ErrorInfo.kt
+++ b/app/src/main/java/org/schabi/newpipe/report/ErrorInfo.kt
@@ -5,7 +5,7 @@ import androidx.annotation.StringRes
 import kotlinx.android.parcel.Parcelize
 
 @Parcelize
-class ErrorInfo private constructor(
+class ErrorInfo(
     val userAction: UserAction?,
     val serviceName: String,
     val request: String,

